Formula/libfyaml.rb

```diff
@@ -0,0 +1,50 @@
+class Libfyaml < Formula
+  desc "Fully feature complete YAML parser and emitter"
+  homepage "https://github.com/pantoniou/libfyaml"
+  url "https://github.com/pantoniou/libfyaml/archive/refs/tags/v0.7.12.tar.gz"
+  sha256 "864e60dc1863e2ba2048b9765d9addd653f6c387e835438f09cb5f26a2a352e1"
+  license "MIT"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gcc" => :build
+  depends_on "libtool" => :build
```

Comment:
`autoconf`, `automake`, and `libtool` are usually for generating configure, so are probably unnecessary with other source tarball. `gcc` should not be needed and will not be used in current state as Homebrew's shims will select Apple Clang on macOS and GCC 11 on Linux.
```suggestion
```